### PR TITLE
Update new-theme package.json and npm modules

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | We wants to update the package.json and webpack
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | You should be able to build assets or use the watch mode, and nothing should break on migrated pages on the BO

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17420)

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/theme.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // Tools
 @import "~prestakit/dist/css/bootstrap-prestashop-ui-kit";
-@import "~bourbon/app/assets/stylesheets/bourbon";
+//@import "~bourbon/app/assets/stylesheets/bourbon";
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/theme.scss]: Please remove comment

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Please remove swap file :  admin-dev/themes/new-theme/.webpack.config.js.swp

**Review by Progi1984 — APPROVED** [review]

File: admin-dev/themes/new-theme/package.json [hunk 2]
```diff
@@ -5,80 +5,90 @@
   "private": "true",
   "scripts": {
     "test": "./node_modules/.bin/mocha  --require babel-register --reporter spec \"tests/**/*.spec.js\"",
-    "build": "NODE_ENV=production webpack --progress --colors --debug --display-chunks",
-    "dev": "webpack --progress --colors --debug --display-chunks --watch",
-    "watch": "npm run dev"
+    "build": "webpack --progress --colors --debug --display-chunks --mode=production",
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/package.json:8]: Maybe we can drop the `--debug` and `--display-chunks` in prod mode now that we're at it, don't you think? It might accelerate things and reduce the noise for people who just want to compile assets. If people want to debug, then they should use the dev mode.

File: admin-dev/themes/new-theme/package.json [hunk 3]
```diff
@@ -5,80 +5,90 @@
...
+    "lint-fix": "eslint -c .eslintrc.js --ext .js,.vue . --fix"
   },
   "author": "PrestaShop",
   "license": "OSL-3.0",
   "dependencies": {
-    "bootstrap-slider": "^6.0.16",
+    "bootstrap-slider": "^10.6.2",
     "bootstrap-tokenfield": "^0.12.0",
-    "bourbon": "^4.2.6",
+    "bourbon": "^6.0.0",
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/package.json:19]: Watch out, lots of things have been dropped in bourbon! This may be a breaking change.

File: admin-dev/themes/new-theme/package.json [hunk 4]
```diff
@@ -5,80 +5,90 @@
...
     "compass": "^0.1.1",
     "dropzone": "^5.5.1",
     "eonasdan-bootstrap-datetimepicker": "^4.17.47",
+    "events": "^3.0.0",
     "fos-routing": "0.0.6",
-    "jets": "0.11.0",
+    "jets": "0.14.1",
     "jquery-ui-dist": "^1.12.1",
     "jwerty": "^0.3.2",
+    "lodash": "^4.17.11",
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/package.json:28]: Why did you add lodash?

File: admin-dev/themes/new-theme/package.json [hunk 5]
```diff
@@ -5,80 +5,90 @@
...
     "lodash.escaperegexp": "^4.1.2",
     "magnific-popup": "^1.1.0",
-    "mobile-detect": "^1.4.3",
+    "mobile-detect": "^1.4.4",
     "moment": "^2.24.0",
-    "perfect-scrollbar": "^1.4.0",
+    "perfect-scrollbar": "^1.5.0",
+    "postcss-cssnext": "^2.10.0",
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/package.json:34]: Is this needed?

File: admin-dev/themes/new-theme/package.json [hunk 6]
```diff
@@ -5,80 +5,90 @@
...
-    "vue": "^2.6.7",
+    "url-polyfill": "^1.1.7",
+    "vue": "^2.6.11",
     "vue-resource": "^1.5.1",
-    "vue-router": "^2.8.1",
-    "vue-template-compiler": "^2.6.7",
-    "vuex": "^2.5.0"
+    "vue-router": "^3.1.5",
+    "vue-template-compiler": "^2.6.11",
+    "vuex": "^3.1.2"
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/package.json:44]: Same comment

File: admin-dev/themes/new-theme/package.json [hunk 7]
```diff
@@ -5,80 +5,90 @@
...
     "typeahead.js": "^0.11.1",
-    "url-polyfill": "^1.1.3",
-    "vue": "^2.6.7",
+    "url-polyfill": "^1.1.7",
+    "vue": "^2.6.11",
     "vue-resource": "^1.5.1",
-    "vue-router": "^2.8.1",
-    "vue-template-compiler": "^2.6.7",
-    "vuex": "^2.5.0"
+    "vue-router": "^3.1.5",
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/package.json:42]: Any reason to update this to the next major version?

File: admin-dev/themes/new-theme/package.json [hunk 8]
```diff
@@ -5,80 +5,90 @@
...
     "babel-polyfill": "^6.26.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.7.0",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-preset-stage-2": "^6.22.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
-    "bootstrap": "~4.0.0",
+    "bootstrap": "~4.4.1",
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/package.json:57]: This should be done at the same time as in the ui kit IMO

**Review by eternoendless — CHANGES_REQUESTED** [review]
